### DISCUSSION TITLE
Where to get types for MDXJsxTextElement and MDXJsxFlowElement

### DISCUSSION THREAD

**Discussion by stefanprobst** [mdx-js/mdx#1796] [Q&A] [answered]
hi, i am currently importing typings for `MDXJsxTextElement` and `MDXJsxFlowElement` from `hast-util-to-estree`, but was wondering if there was an official mdx types package for this?

**Comment by wooorm** [comment, marked as answer]
They're in https://github.com/syntax-tree/mdast-util-mdx. They might be exported from remark-mdx maybe?

**Reply by stefanprobst** [reply]
thanks!
hmm yes i think `remark-mdx` makes sense (or a separate `@types/mdxast` package?)
